microsoft/testsuites/network/sriov.py

```diff
@@ -596,24 +596,27 @@ def verify_sriov_ethtool_offload_setting(self, environment: Environment) -> None
             ).is_equal_to(False)
 
         # reload sriov modules
-        module_in_used = ""
-        for node in environment.nodes.list():
-            module_in_used = remove_module(node)
-        for node in environment.nodes.list():
-            load_module(node, module_in_used)
+        module_built_in = False
```

Comment:
You don't need this line since the value will be set to 'True' or 'False' in the next line.

Reply:
fixed